frontend/public/locales/fr/personal-information.json

```diff
@@ -99,7 +109,7 @@
       "change": "(FR) Change"
```

Comment:
**Suggestion:** With `_gcweb-app.personal-information.preferred-language._index.tsx` deleted, I feel these translations can be removed as well.

Reply:
By "these translations", i mean all translations within `index`.